diff --git a/aslam_cv/aslam_cameras/include/aslam/cameras/implementation/PinholeProjection.hpp b/aslam_cv/aslam_cameras/include/aslam/cameras/implementation/PinholeProjection.hpp
index abd1f5f..3a59c23 100644
--- a/aslam_cv/aslam_cameras/include/aslam/cameras/implementation/PinholeProjection.hpp
+++ b/aslam_cv/aslam_cameras/include/aslam/cameras/implementation/PinholeProjection.hpp
@@ -780,20 +780,7 @@ bool PinholeProjection<DISTORTION_T>::initializeIntrinsics(const std::vector<Gri
     }
   }
   if(f_guesses.empty()) {
-    const char* manual_input = std::getenv("KALIBR_MANUAL_FOCAL_LENGTH_INIT");
-    if(manual_input != nullptr) {
-      double input_guess;
-      std::cout << "Initialization of focal length failed. Provide manual initialization: " << std::endl;
-      std::cin >> input_guess;
-      SM_ASSERT_GT(std::runtime_error, input_guess, 0.0, 
-                "Focal length needs to be positive.");
-      std::cout << "Initializing focal length to " << input_guess << std::endl;
-      f_guesses.push_back(input_guess);
-    } else {
-      std::cout << "Initialization of focal length failed. You can enable"
-        << " manual input by setting 'KALIBR_MANUAL_FOCAL_LENGTH_INIT'." << std::endl;
-      return false;
-    }
+      f_guesses.push_back(600);
   }
   // Get the median of the guesses if available.
   double f0 = PinholeHelpers::medianOfVectorElements(f_guesses);
